quarkchain/utils.py

```diff
@@ -73,28 +76,104 @@ def assert_true_with_timeout(f, duration=1):
     asyncio.get_event_loop().run_until_complete(d())
 
 
+_LOGGING_FILE_PREFIX = os.path.join("logging", "__init__.")
+
+
+class QKCLogger(logging.getLoggerClass()):
+    """https://github.com/abseil/abseil-py/blob/master/absl/logging/__init__.py
+    refer to ABSLLogger
+    """
+
+    def findCaller(self, stack_info=False):
```

Comment:
not see it being used. is it used internally by the logging module?

Reply:
yeah, `override`